MD/md-38/README.md

```diff
@@ -0,0 +1,76 @@
+# MD-38: Provide for Fixed Token Supply when Using Atomic Bridge and Fast Finality Settlement (FFS)
+- **Description**: Requests for Movement Improvement Proposals (MIPs) to provide for fixed token supply when using the Atomic Bridge and Fast Finality Settlement (FFS).
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: Andreas Penzkofer
+
+
+## Overview
+![Fixed Supply Rewarding Problem](./fixed-supply-rewarding-problem.png)
```

Comment:
- this figure is displayed without any description or context. it is not referenced in the text anywhere.
- at no point burning is mentioned in this file
- There is no reference to the model that describes the DEX approach. also for the sake of this MD and the description of the burn problem the DEX is not necessary. I suggest to add a figure that has a simple direct bridge from L1MOVE <> L2MOVE, and where L2MOVE is the gas. Such a model seems to be minimal yet envelop the problem sufficiently.